app/qml/components/MMHeader.qml

```diff
@@ -73,6 +81,35 @@ Item {
     onClicked: root.backClicked()
   }
 
+  Rectangle {
```

Comment:
This is not correct approach. Personal Icon should be bound from parent as right item

Reply:
We can discuss it. This is a special item and should be extended by the new design, it can contain a red mark on the top right corner of this icon or can contain user initials instead of the icon.